experimental/packages/otlp-transformer/src/trace/index.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { InstrumentationLibrary } from '@opentelemetry/core';
+import type { Resource } from '@opentelemetry/resources';
+import type { ReadableSpan } from '@opentelemetry/sdk-trace-base';
+import { toAttributes } from '../common/internal';
+import { toSpan } from './internal';
+import { IExportTraceServiceRequest } from './types';
+
+export function createExportTraceServiceRequest(spans: ReadableSpan[]): IExportTraceServiceRequest | null {
+  if (spans.length === 0) {
+    return null;
+  }
+
+  return {
+    resourceSpans: groupByResource(spans).map(({ resource, resourceSpans, schemaUrl: resourceSchemaUrl }) => ({
+      resource: {
+        attributes: toAttributes(resource.attributes),
+        droppedAttributesCount: 0,
+      },
+      instrumentationLibrarySpans: resourceSpans.map(({ instrumentationLibrary, instrumentationLibrarySpans, schemaUrl: librarySchemaUrl }) => ({
+        instrumentationLibrary: {
+          name: instrumentationLibrary.name,
+          version: instrumentationLibrary.version,
+        },
+        spans: instrumentationLibrarySpans.map(toSpan),
+        schemaUrl: librarySchemaUrl,
+      })),
+      schemaUrl: resourceSchemaUrl,
+    }))
+  };
+}
+
+type ResourceSpans = {
+  resource: Resource,
+  resourceSpans: InstrumentationLibrarySpans[],
+  schemaUrl?: string,
```

Comment:
This value is not being populated by the code below. However, it is found in the protos. Do we need to add it to the SDK anywhere? Is there an open issue for that?

Reply:
it's missing in our sdk impl right now I think. i'll add it to this so it doesn't need to be added later.